src/GoTrueClient.ts

```diff
@@ -1229,13 +1242,37 @@ export default class GoTrueClient {
     if (options?.scopes) {
       urlParams.push(`scopes=${encodeURIComponent(options.scopes)}`)
     }
+    if (options?.isPKCE) {
+      const codeVerifier = this._generatePKCECodeVerifier()
+      setItemAsync(this.storage, this.storageKey, codeVerifier)
```

Comment:
Just realizing I'm not a fan of this... given that `setItemAsync` is cookie storage in auth helpers. :/